test/README.md

```diff
@@ -17,10 +17,18 @@ _By default `go test` will not run [the e2e tests](#running-end-to-end-tests), w
 
 ## Running end to end tests
 
+### Environment requirements
```

Comment:
This is *duplicate* with https://github.com/knative/build/pull/407/files#diff-c30ff793110b0960f76bb3854af6f5a8R56, we should merge those.

Reply:
oh, i am negligent and do some changes. PTAL